app/controllers/orders_controller.rb

```diff
@@ -1,4 +1,78 @@
 # frozen_string_literal: true
 
 class OrdersController < ApplicationController
+  before_action :parse_cart_data, only: :create
+  before_action :find_products, only: :create
+
+  def index
+    @orders = Order.all
```

Comment:
```suggestion
    @orders = Order.all
```
Chỗ này hiển thị list order của user hiện tại thôi. .all là hiển thị hết tất cả order rồi. Nhớ sắp xếp theo mới nhất. Có cần pagy không